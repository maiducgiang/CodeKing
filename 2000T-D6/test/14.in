45
21560 19672 59819 37500 29742 17627 75433 47183 45892 56483 52912 76636 8169 17230 16289 99682 89123 21543 47013 24180 29081 25386 58764 31995 43340 20570 8558 47469 28250 5781 29189 46222 33335 99055 74766 13172 94814 21770 15000 24256 45784 4578 64275 29397 62651 