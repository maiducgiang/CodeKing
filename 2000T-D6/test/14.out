2695 2459 1459 9375 14871 -1 313 1627 11473 8069 3307 19159 2723 8615 2327 49841 -1 7181 15671 6045 2237 12693 14691 395 10835 10285 4279 15823 14125 1927 101 23111 6667 19811 37383 3293 47407 10885 1875 379 5723 2289 21425 9799 2021 
2 2 41 2 2 -1 241 29 2 7 2 2 3 2 7 2 -1 3 3 2 13 2 2 3 2 2 2 3 2 3 17 2 5 5 2 2 2 2 2 2 2 2 3 3 31 